test_session_mgmt.py

```
@@ -0,0 +1,536 @@
+from pathlib import Path
+from unittest import mock
+
+HERE = Path(__file__).resolve().parent
+SERVER_PATH = HERE / "mcp_server.py"
+
+# --- Synthetic secret-shape fixtures (assembled at runtime) ---------------
+_GOOG_API_KEY_PREFIX = "AI" + "za"
+FAKE_GOOG_API_KEY = _GOOG_API_KEY_PREFIX + "SyDsyntheticTestValue1234567890_-end"
+
+
+def _stub_module(name: str, **attrs) -> types.ModuleType:
```

Ruff (0.15.12)

[warning] Missing type annotation for `**attrs`

(ANN003)